charts/router/templates/router-clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+{{ if .Capabilities.APIVersions.Has (include "rbacApiVersion" .) }}
+kind: ClusterRole
+apiVersion: {{ template "rbacApiVersion" . }}
+metadata:
+  name: deis:deis-router
```

Comment:
Is this colon allowed here? Even if so, I would consider changing this name. My understanding is that names of most things in k8s should look like valid DNS names, which I believe excludes colons. Even if it's technically permissible in this particular spot, I would rather we still deferred to that convention.